public/gameInfo.php

```diff
@@ -120,11 +120,23 @@
 $totalEarnedTrueRatio = 0;
 $totalPossibleTrueRatio = 0;
 
-$authors = [];
+$authorName = Array();
+$authorCount = Array();
```

Comment:
As we're running PHP 7.1+ please use the `[]` notation instead of `Array()`

Reply:
Ok, I'll remember that going forward.